tools/system_libs.py

```diff
@@ -68,26 +68,25 @@ def get_cflags(force_object_files=False):
   return flags
 
 
-def run_build_command(cmd):
-  # this must only be called on a standard build command
-  assert cmd[0] == shared.PYTHON and cmd[1] in (shared.EMCC, shared.EMXX)
-  # add standard cflags, but also allow the cmd to override them
-  cmd = cmd[:2] + get_cflags() + cmd[2:]
+def _run_one_command(cmd):
+  # Helper function used by run_build_commands
+  if shared.EM_BUILD_VERBOSE:
+    print(' '.join(cmd))
   shared.run_process(cmd, stdout=stdout, stderr=stderr)
 
 
-def run_commands(commands):
+def run_build_commands(commands):
```

Comment:
maybe this should also have a `_` prefix to clarify it too is an internal thing?

Reply:
Actually I think I'll revert remove the `_`.  If we are going to apply that rule it should be consistent and done as a followup.